Sources/SharedModels/User/Account.swift

```diff
@@ -35,6 +35,21 @@ public struct Account: Codable {
 
         return userAuthorities.contains(authority)
     }
+
+    public static let mock = Account(
+        id: 1,
+        login: "artemis",
+        name: "Artemis User",
```

Comment:
Please use personas like John Doe instead. I think there is also a confluence page for that: https://confluence.ase.in.tum.de/display/ArTEMiS/Random+Personas

Reply:
Thanks, didn't know we had something like this. Fixed!